swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/stable/2021-01-31-preview/examples/OpenShiftClusters_CreateOrUpdate.json

```diff
@@ -0,0 +1,162 @@
+{
```

Comment:
I think these files should be going into swagger/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/preview/2021-01-31-preview